docs/source/DeveloperDocumentation/PlayerIntegrations/videojs.md

```diff
@@ -0,0 +1,66 @@
+# Video JS
+
+[Video JS](https://videojs.com) is an industry leading HTML5 video player.
+
+The [vuplay-videojs](https://github.com/vualto/vuplay-videojs) repository demonstrates at a lower level how to integrate [VUDRM](https://docs.vualto.com/projects/vudrm/en/latest/index.html) with Video JS.
+
+## Example of Video JS Integration
+
+You will need to load the following libraries:
+
+```html
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/video.js/7.10.2/video.min.js"></script>
+    <script src="https://unpkg.com/@videojs/http-streaming@2.3.0/dist/videojs-http-streaming.js"></script>
+    <script src="https://unpkg.com/videojs-contrib-eme@3.7.0/dist/videojs-contrib-eme.js"></script>
+```
+
+The following code snippet can be found in the repo mentioned above here: [https://github.com/Vualto/vuplay-videojs/blob/master/src/vuplay.js](https://github.com/Vualto/vuplay-videojs/blob/master/src/vuplay.js)
+
+```javascript
+const streamURL = "<your-stream-url>"
+const contentId = "<content-id>";
+
+const token = encodeURIComponent(
+    "<your-vudrm-token>"
+);
+
+// playready
+const playReadyLicenseServerURL =
+    "https://playready-license.vudrm.tech/rightsmanager.asmx?token=" + token;
+
+// widevine
+const widevineLicenseServerURL =
+    "https://widevine-license.vudrm.tech/proxy?token=" + token;
+
+// fairplay
+const fairplayCertificateUri =
+    "https://fairplay-license.vudrm.tech/certificate/vualto-demo";
```

Comment:
Not a biggy but might be a good shout to make it super clear that this URL should have their client name on the end.
```suggestion
    "https://fairplay-license.vudrm.tech/certificate/<your-client-name>";
```